### ISSUE TITLE
Messages being sent incorrectly

### ISSUE THREAD

**Issue by carlbeattie2000** [carlbeattie2000/playrust_discord_bot#1]
Connection to the rust server's web socket is being made, messages can be sent and received. The issue occurs on every message being sent, it's returning a message with an error `"not_found"`. I'm unable to find what's causing the issue, although I assume the player token is incorrect, as I faced the same issue when I tested on the original library I've converted for TS (Which is what made me believe for 5 hours I had messed up somewhere, given I've only started learning about Protocol Buffers today, and was not using the same method as the library.). I still believe I've still messed up somewhere, but I don't believe it's within the service handling the web socket connection.

I'm assuming I've made a mistake with the steam+rust connection, which is giving me wrong details. Likewise, I've noticed the player Token is always negative, and I'm unsure if it should be. Going to try and watch the data being sent from the game tomorrow and see if it matches what I'm receiving on the app.

Once this bug has been ironed out, I can rewrite the websocket+pairing flow to handle multiple servers instead of just the one.

Example message being sent: 
```
{
  seq: 1,
  playerId: 000000000000,
  playerToken: -000000000000,
  entityId: 0,
  getInfo: undefined,
  getTime: undefined,
  getMap: undefined,
  getTeamInfo: undefined,
  getTeamChat: undefined,
  sendTeamMessage: { message: '[BOT]: RustPlusBot connected!' },
  getEntityInfo: undefined,
  setEntityValue: undefined,
  checkSubscription: undefined,
  setSubscription: undefined,
  getMapMarkers: undefined,
  promoteToLeader: undefined,
  getClanInfo: undefined,
  setClanMotd: undefined,
  getClanChat: undefined,
  sendClanMessage: undefined,
  getNexusAuth: undefined,
  cameraSubscribe: undefined,
  cameraUnsubscribe: undefined,
  cameraInput: undefined
}
```

The response:
```
{
  response: {
    seq: 1,
    success: undefined,
    error: { error: 'not_found' },
    info: undefined,
    time: undefined,
    map: undefined,
    teamInfo: undefined,
    teamChat: undefined,
    entityInfo: undefined,
    flag: undefined,
    mapMarkers: undefined,
    clanInfo: undefined,
    clanChat: undefined,
    nexusAuth: undefined,
    cameraSubscribeInfo: undefined
  },
  broadcast: undefined
}
```

**Comment by carlbeattie2000** [comment]
Notes I found from scrolling a year back in the library's discord server:
```
if you get the error: not_found, that means the entity does not exist. But, it also can mean that you don't have TC auth for that specific entity or that your playerToken is invalid.
so if you can fetch server info with your player token, you know that the playerToken is fine, and it's either tc auth or the entity doesn't exist.
```

I think the best move would be to open a new branch and add the functionality to handle multiple server connections, with room for a function to later remove/refresh/repair connections. This will eliminate the possible issue of different servers using the wrong token. Once I've done this, the next step will be validating js is handling the playerid correctly and not chopping it up. I read the original library is passing it as a string to avoid this issue. Should also note, that I think I should store the ID of the notification to prevent re-sending every server boot.

**Comment by carlbeattie2000** [comment]
Fixed in b05eb6f1b1b48e430a805fa8540c62f377fa3f12
